packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -329,6 +329,11 @@ class DurationRange {
 
 /// [VideoPlayerOptions] can be optionally used to set additional player settings
 class VideoPlayerOptions {
+  /// Set this to false to let plugin manage play/pause with app lifecycle changes,
```

Comment:
Remove this sentence; you don't generally need to document both options for a bool since they are inverse of each other.